geth/whisper/request_historic_messages.go

```diff
@@ -70,118 +74,143 @@ func RequestHistoricMessagesHandler(nodeManager common.NodeManager) rpc.Handler
 }
 
 type historicMessagesRequest struct {
-	Peer     []byte              //mailbox peer
-	TimeLow  uint32              //resend messages from
-	TimeUp   uint32              //resend messages to
-	Topic    whisperv5.TopicType //resend messages by topic
-	SymkeyID string              //Mailbox symmetric key id
-	PoW      float64             //whisper proof of work
-}
+	// MailServer enode address.
+	Peer []byte
 
-func parseArgs(args ...interface{}) (historicMessagesRequest, error) {
-	var (
-		r = historicMessagesRequest{
-			TimeLow: uint32(time.Now().Add(-24 * time.Hour).Unix()),
```

Comment:
Could we save that code? It does exactly the same as `r.TimeUp = uint32(time.Now().UTC().Unix())` from:
```
if t, ok := param["to"]; ok {
	if value, ok := parseToUint32(t); ok {
		r.TimeUp = value
	} else {
 		return r, fmt.Errorf("to value must be unix time in seconds, got: %T", t)
 	}
 } else {
 	r.TimeUp = uint32(time.Now().UTC().Unix())
 }
```
but it's easier to read.